java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/docker/StageDockerArtifactBuildStep.java

```diff
@@ -90,6 +95,23 @@ protected void doBuild(Path directory, Map<String, String> metadata) throws Buil
   }
 
   /*
+   * Deletes a directory and all of its contents.
+   */
+  private void deleteDirectoryRecursive(Path path) throws IOException {
```

Comment:
It's up to you. I don't mind re-adding it. If you decide to keep this method, you should write a test for it that exercises it directly. Also, the method should be public static and moved to a Utils class or something.